### ISSUE TITLE
Review and update .gov accessibility plan

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/manage.get.gov#2207]
### Issue description

On a quarterly basis, Product needs to review and update the Section 508 Compliance accessibility plan.

### Acceptance criteria

- [x] Accessibility plan has been reviewed and updated as needed
- [x] The accessibility quarterly scorecard has been reviewed and updated as needed
- [x] Accessibiility certification process initiated, if necessary and appropriate. (Tickets created for .gov and OAST)
- [x] New ticket for the next quarter's document review is created.

### Additional context

[Accessibility Plan](https://docs.google.com/document/d/1Rt6XN7yZKvdA3A2KRkv54SzmkjQ4jOb6Xknhd0mOvKE/edit)
[Quarterly scorecard](https://docs.google.com/spreadsheets/d/1aVZY7nLdeDHYoQnY2y-uGbD16LV9vN-ZPgMNhpfh0YM/edit?usp=sharing)

### Links to other issues

Relates to:
- #1917 
Blocked by:
- #2341

**Comment by rileyorr** [comment]
@gabydisarli I added an AC capturing the accessibility quarterly scorecard needing to be reviewed as well. Thank you for calling that out!

**Comment by PaulKuykendall** [comment]
https://pra.digital.gov/about/
In case we wish to launch an Opt-in campaign.

**Comment by AnnaGingle** [comment]
@PaulKuykendall Are you able to create the ticket for next quarter so we can move this into done?
